src/data/links.yaml

```diff
@@ -1443,6 +1443,40 @@ links:
       transacted and advertised in the retail market, protecting both the buyer
       and the seller.
 
+  - title: TenTree
+    url: https://www.tentree.ca/
+    featured: true
+    categories:
+      - clothing
+    tags: [clothing]
+    description: >-
+      Ten tree is a brand that actively promotes environmental stewardship. They have a wide variety of comfortable clothes and accessories and with each purchase, they pledge to plant 10 trees.
+
+  - title: Arvin  Goods
+    url: https://www.arvingoods.com/
+    featured: true
+    categories: 
+      - clothing
+    tags: [clothing]
+    description: >-
+      Brand that makes fashionable socks through a sustainable supply chain process they have developed. They have removed excess water from their manufacturing process - leaving more water left for communities to drink!
+
+  - title: Siizu
+    url: https://siizu.com/
```

Comment:
I can't find much information other than they say the words Ethical and Sustainable a lot. This kinda feels like greenwashing to me.

Reply:
Although I agree that there is some greenwashing* going on with Siizu, they do actually do some cool things.  I think it is a good resource regardless, for the following reasons:

- They make a concerted effort to manufacture materials in factories very close to where the raw material is sourced, 'no flying necessary'
- Make sea wool fabric that is made from oyster shell waste!  This is the first I've heard of this.
- Shipped in recyclable, plastic-free packaging

Let's celebrate the companies trying to do better, even if they're also doing a little bit of greenwashing too.  

*greenwashing examples: 'We're organic' (according to whom?), '100% sustainably grown' (not the same as organic, what does this even mean), '0% toxins' (what about the dyes?), "SiiZU donates generously to help American Forest reach its goal of planting 2.7 million trees"  - what do they donate, how much, is it per purchase, per dollar, or have they only donated one tree? Why don't they share this info?"

Reply:
@philsturgeon, I think this is good to merge.

Reply:
Just adding the new link and updated editorial. Ill push it shortly!

Reply:
@jmcdona1d I see another commit got pushed - that all for the links here? We good to go?

Reply:
@HeidiHansen all great points thank you! Can you email them that feedback, they might not know they’re being too vague.